ui/src/app/pages/appAnalysis/app-files-list.tsx

```diff
@@ -104,7 +104,12 @@ export const AppFilesList = withCurrentWorkspace()(
       // Find App Type on the basis of file name extension
       const { name } = row;
       const appType = getAppInfoFromFileName(name).appType;
-      return <AppLogo appType={appType} style={{ marginRight: '1em' }} />;
+      return (
+        <AppLogo
+          appType={appType}
+          style={{ marginRight: '1em', maxHeight: '4.5rem' }}
```

Comment:
Is this an attempt to make the SAS rows the same height as the others?  It's not working for me - SAS is still too big.  The problem looks like the height of the icon.

Let's revert this change for now and merge this PR.  We can fix the icon styling in a later PR.

Reply:
ah, okay. Will revert now. It works for my browser. But I also notice the table size is relative and I get different size using different size of browser window. Hopefully there can be a better fix...